scripts/stats/global_ens/exevs_global_ens_atmos_grid2grid_stats.sh

```diff
@@ -74,11 +74,11 @@ if [ $ens = all ] || [ $ens = gefs ] || [ $ens = cmce ] || [ $ens = naefs ] || [
 
      elif [ $ens = ecme ] && [ $verif_case = upper ] ; then
 
-      if [ ! -s ${COMIN}.${VDATE}/cmce/cmcanl.t00z.grid3.f000.grib2 ] ; then
+      if [ ! -s ${EVSIN}.${VDATE}/cmce/cmcanl.t00z.grid3.f000.grib2 ] ; then
```

Comment:
@BinbinZhou-NOAA , can you correct this line to be a check for ecme data? I think it should be:
```
if [ ! -s ${EVSIN}.${VDATE}/ecme/ecmanl.t00z.grid3.f000.grib2 ] ; then
```